cygnus-ngsi/conf/agent_ngsi.conf.template

```diff
@@ -386,7 +386,7 @@ cygnus-ngsi.sinks.cartodb-sink.channel = cartodb-channel
 # absolute path to the CartoDB file containing the mapping between FIWARE service/CartoDB usernames and CartoDB API Keys
 cygnus-ngsi.sinks.cartodb-sink.keys_conf_file = /usr/cygnus/conf/cartodb_keys.conf
 # if true the latitude and longitude values are exchanged, false otherwise
-#cygnus-ngsi.sinks.cartodb-sink.flip_coordinates = true
+#cygnus-ngsi.sinks.cartodb-sink.swap_coordinates = true
```

Comment:
Potential backward compatibility problem here? Or this feature has been never released?